lib/elixir/lib/registry.ex

```diff
@@ -585,7 +594,8 @@ defmodule Registry do
       [{self(), {1, :atom, 1}}, {self(), {2, :atom, 2}}]
       iex> Registry.match(Registry.MatchTest, "hello", {:_, :_, :"$1"}, [{:>, :"$1", 1}])
       [{self(), {2, :atom, 2}}]
-      iex> Registry.match(Registry.MatchTest, "hello", {:_, :"$1", :_}, [{:is_atom, :"$1"}]) |> Enum.sort()
```

Comment:
It was ~ 2 characters too wide 😱
https://hexdocs.pm/elixir/master/Registry.html#match/4-examples